[BUG] openmetadata.config.elasticsearch.payLoadSize rendered incorrectly

### Checks

- [X] I have checked for existing issues.
- [X] This report is about the Openmetadata Helm Chart.
- [ ] This report is about the Openmetadata Dependencies Helm Chart.

### Description

The newly introduced variable with chart version `v1.4.0` is being rendered incorrectly by Helm when applying the chart. Due to that, OpenMetadata fails to start up properly.  

The default value for `openmetadata.config.elasticsearch.payLoadSize` is `10485760`. Helm templates it out like this into the secret `openmetadata-search-secret`:

```shell
~ helm template openmetadata open-metadata/openmetadata -n openmetadata --dry-run=client|grep ELASTICSEARCH_PAYLOAD_BYTES_SIZE
  ELASTICSEARCH_PAYLOAD_BYTES_SIZE: IjEuMDQ4NTc2ZSswNyI=                                                                                                                                                                                     
~ echo "IjEuMDQ4NTc2ZSswNyI="|base64 -d
"1.048576e+07"%
```

If you change the secret variable manually within K8s, OpenMetadata starts up properly. So in the end there seems to be something wrong when templating this value out in Helm.

### Steps to reproduce

1. Take chart version 1.4.0 and install it with the default values (or upgrade and existing release)
2. OpenMetadata pod will fail to start up due to the incorrect value:

```log
   |||||||
 ||||   ||||      ____
||||     ||||    / __ \
||||     ||||   | |  | | _ __    ___  _ __
|||||   |||||   | |  | || '_ \  / _ \| '_ \
|||||||||||||   | |__| || |_) ||  __/| | | |
|||||||||||||    \____/ | .__/  \___||_| |_|
||| ||||| |||    __  __ | |    _              _         _
|||  |||  |||   |  \/  ||_|   | |            | |       | |
|||   |   |||   | \  / |  ___ | |_  __ _   __| |  __ _ | |_  __ _
|||       |||   | |\/| | / _ \| __|/ _` | / _` | / _` || __|/ _` |
||| || || |||   | |  | ||  __/| |_| (_| || (_| || (_| || |_| (_| |
||| ||||| |||   |_|  |_| \___| \__|\__,_| \__,_| \__,_| \__|\__,_|
 |||||||||||
   |||||||
Migrating the OpenMetadata Schema.
Failed to db migration due to 
io.dropwizard.configuration.ConfigurationParsingException: /opt/openmetadata/bootstrap/../conf/openmetadata.yaml has an error:
  * Incorrect type of value at: elasticsearch.payLoadSize; is of type: String, expected: Integer

...

Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.lang.Integer` from String "1.048576e+07": not a valid `java.lang.Integer` value
```

### Chart Version

1.4.0

### App Version

1.4.0

### Kubernetes Version

1.29.4

### Helm Version

3.15.0

### Relevant Logs

```shell
Log already specified in the "how to reproduce" section.
```


### Custom Helm Values

_No response_

### Have you joined Openmetadata Slack community?

No

--- a/charts/openmetadata/Chart.yaml
+++ b/charts/openmetadata/Chart.yaml
@@ -16,7 +16,7 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
-version: 1.4.0
+version: 1.4.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to

--- a/charts/openmetadata/templates/secrets.yaml
+++ b/charts/openmetadata/templates/secrets.yaml
@@ -47,7 +47,7 @@ data:
   ELASTICSEARCH_INDEX_MAPPING_LANG: {{ .searchIndexMappingLanguage | quote| b64enc }}
   ELASTICSEARCH_KEEP_ALIVE_TIMEOUT_SECS: {{ .keepAliveTimeoutSecs | quote | b64enc }}
   ELASTICSEARCH_CLUSTER_ALIAS: {{ .clusterAlias | quote | b64enc }}
-  ELASTICSEARCH_PAYLOAD_BYTES_SIZE: {{ .payLoadSize | quote | b64enc }}
+  ELASTICSEARCH_PAYLOAD_BYTES_SIZE: {{ .payLoadSize | int | toString | b64enc }}
   {{- if .trustStore.enabled }}
   ELASTICSEARCH_TRUST_STORE_PATH: {{ .trustStore.path | b64enc }}
   {{ end }}

